### DISCUSSION TITLE
Fruits Delight

### DISCUSSION THREAD

**Discussion by Lyxairo** [DynamicTreesTeam/DynamicTrees#966] [Compatibility Addon Suggestions]
I'd like to request an addon for Fruits Delight by Icy0x1 which gives new fruit trees
https://www.curseforge.com/minecraft/mc-mods/fruits-delight
Preferably for 1.20.1
(There are seven sapplings)

![pic1](https://github.com/user-attachments/assets/4e30b54b-03e3-4409-8022-55536c309d5c)
![pic2](https://github.com/user-attachments/assets/edc0bd25-ecf8-4c2c-852a-7381fbb0d1de)

**Comment by TravelerMist** [comment]
Please can this addon be made? I have a modpack and this is one of the only mods to need dynamic trees support.

**Comment by Groupix05** [comment]
https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-fruits-delight
It's in beta rn because some features are not working at the moment

**Comment by valaDT** [comment]
Hi ! Would it be possible to eventually update towards 1.21.1 neoforge? We use dynamic tree and fruits in our pack! 
thank you in advance if it’s possible that would be great!

**Reply by Groupix05** [reply]
Yes, but not soon because im taking a break from Dynamic trees. You can update the addon yourself if you really need, and make a pull request on the github.

**Reply by valaDT** [reply]
Okay thanks, i gonna see what i can do with it !

**Reply by Igiuriel** [reply]
ValaDT, Have you had any success updating the addon?

**Reply by valaDT** [reply]
Hello, no, i haven't found someone who know how to do it :/

**Comment by BrandonPotter0** [comment]
Oooo, Any chance of a 1.21.1 - NeoForge Version as well? :D

**Comment by reldred** [comment]
Would love to see this updated to 1.21.1 🙏

**Comment by Enderlais** [comment]
imma bump it for 1.21.1 neoforge

**Comment by Gigiboys** [comment]
Hi. I just downloaded the mod in 1.20.1 but can't seem to get any fruit by right-clicking on the leaves or breaking the tree. Is it possible to have a way to collect fruits, possibly without breaking the tree ?

**Reply by Groupix05** [reply]
With dynamic trees for fruits trees, you need to wait for the actual fruit to grow on the tree. Same for dynamic trees pams harvestcraft, croptopia or fish of thieves, and some others. If you use a season mod, like serene seasons, you need to be in the correct season for the fruit to grow,

**Reply by Gigiboys** [reply]
You added this to fruits delight ? That's awesome thanks I thought it was discontinued after 1.12 !
(it's a bit long tho, I'll be patient for the harvest)

**Comment by TakehisaYuji** [comment]
we fr need this on 1.21.1

**Reply by Groupix05** [reply]
1.21.1 version is now on CF
